packages/shell-api/src/database.ts

```diff
@@ -264,8 +272,8 @@ export default class Database extends ShellApiClass {
   getSiblingDB(db: string): Database {
     assertArgsDefined(db);
     this._emitDatabaseApiCall('getSiblingDB', { db });
-    if (this._baseOptions.session) {
-      return this._baseOptions.session.getDatabase(db);
+    if (this._session) {
```

Comment:
I don't know if there will be any other db-wide options we need to send to every command in the future, but you're right it's better to be more specific now and then just update if we ever need it

Reply:
Ultimately, I was just going for “something that documents that `_baseOptions` actually might contain” here, because I was a bit confused by that in the past :)